src/actions/generate.ts

```diff
@@ -0,0 +1,21 @@
+import { ChainId } from "@iov/bcp";
+import { Bip39, Random } from "@iov/crypto";
+import { UserProfile } from "@iov/keycontrol";
+
+import { codecFromString } from "../codec";
+import { setSecretAndCreateIdentities } from "../profile";
+
+export async function generate(args: ReadonlyArray<string>): Promise<void> {
+  if (args.length < 2) {
+    throw Error(`Not enough arguments for action 'generate'. See README for arguments.`);
+  }
+  const codecName = codecFromString(args[0]);
+  const chainId = args[1] as ChainId;
+
+  const mnemonic = Bip39.encode(await Random.getBytes(16)).asString();
+  console.log(`FAUCET_MNEMONIC="${mnemonic}"`);
+  console.log("");
```

Comment:
What's this for?

Reply:
To produce the empty line you see in the example output I posted in this PR's description